root/etc/cont-init.d/40-plex-first-run

```diff
@@ -136,5 +136,5 @@ if [ -z "$(getPref "TranscoderTempDirectory")" ]; then
   setPref "TranscoderTempDirectory" "/transcode"
 fi
 
-touch /.firstRunComplete
+touch /tmp/.firstRunComplete
```

Comment:
If moving to another dir, this likely would need to run a `mkdir` for that dir first in case it doesn't already exist